Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -111,7 +111,7 @@ RemoteGraphicsContextGLProxy::~RemoteGraphicsContextGLProxy()
 void RemoteGraphicsContextGLProxy::initializeIPC(Ref<IPC::StreamClientConnection>&& streamConnection, RenderingBackendIdentifier renderingBackend, IPC::StreamServerConnection::Handle&& serverHandle)
 {
     m_streamConnection = WTFMove(streamConnection);
-    m_streamConnection->open(*this, *this);
+    protectedStreamConnection()->open(*this, *this);
```

Comment:
Instead of doing this, why not just make a copy in the line above and use `streamConnection` here as in:
```
m_streamConnection = streamConnection;
streamConnection->open(*this, *this);
```